### ISSUE TITLE
Could my language be added?

### ISSUE THREAD

**Issue by aspizu** [alilleybrinker/langs-in-rust#114]
<https://github.com/aspizu/onyo>

**Comment by alilleybrinker** [comment]
Yes! Though it's probably best if you add them yourself using the script provided in the repo. I don't know that I'll have the time soon to do it.
